lib/routes/admin/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+module.exports = function(mediator) {
+  const router = express.Router();
```

Comment:
0.10 only has a problem with `const` in strict mode, we should consider adding the [strict linting rule](http://eslint.org/docs/rules/strict) to enforce strict mode too